config/configgrpc/configgrpc.go

```diff
@@ -226,16 +226,58 @@ func (gcs *ClientConfig) isSchemeHTTPS() bool {
 // a non-blocking dial (the function won't wait for connections to be
 // established, and connecting happens in the background). To make it a blocking
 // dial, use grpc.WithBlock() dial option.
-func (gcs *ClientConfig) ToClientConn(ctx context.Context, host component.Host, settings component.TelemetrySettings, extraOpts ...grpc.DialOption) (*grpc.ClientConn, error) {
-	opts, err := gcs.toDialOptions(ctx, host, settings)
+//
+// Deprecated: [v0.109.0] If providing a [grpc.DialOption], use [ClientConfig.ToClientConnWithOptions]
```

Comment:
This (and similar ones below) need an update since v0.109.0 is out
```suggestion
// Deprecated: [v0.110.0] If providing a [grpc.DialOption], use [ClientConfig.ToClientConnWithOptions]
```